bot_management/bot_manager.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+from bot_management.bot_body import Bot
+import threading
+from bot_management.my_keys import *
```

Comment:
It is not recommended to user implicite imports. Python should be explicite not implicite.
Also why do we store api_keys in a codebase? These information should be handled by REST API and Postgres.

Reply:
It is just for my testing, so i can store keys in separate file when presenting the code. It is reccomendet to not share your keys.